<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>633</width>
    <height>368</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>0</y>
     <width>181</width>
     <height>351</height>
    </rect>
   </property>
   <property name="title">
    <string>Parameters</string>
   </property>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>111</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>13</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Distance Penalty</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="DistancePenalty">
    <property name="geometry">
     <rect>
      <x>125</x>
      <y>30</y>
      <width>49</width>
      <height>24</height>
     </rect>
    </property>
    <property name="maximum">
     <number>255</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>90</y>
      <width>111</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>13</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Population Size</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="PopulationSize">
    <property name="geometry">
     <rect>
      <x>125</x>
      <y>90</y>
      <width>49</width>
      <height>24</height>
     </rect>
    </property>
    <property name="maximum">
     <number>100</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>120</y>
      <width>111</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>13</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Min Code Size</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="MinCodeSize">
    <property name="geometry">
     <rect>
      <x>125</x>
      <y>120</y>
      <width>49</width>
      <height>24</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>150</y>
      <width>111</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>13</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Max Code Size</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="MaxCodeSize">
    <property name="geometry">
     <rect>
      <x>125</x>
      <y>150</y>
      <width>49</width>
      <height>24</height>
     </rect>
    </property>
    <property name="maximum">
     <number>500</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>180</y>
      <width>111</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>13</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Mutation Rate</string>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="MutationRate">
    <property name="geometry">
     <rect>
      <x>125</x>
      <y>180</y>
      <width>49</width>
      <height>24</height>
     </rect>
    </property>
    <property name="singleStep">
     <double>0.001000000000000</double>
    </property>
   </widget>
   <widget class="QLabel" name="label_12">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>240</y>
      <width>111</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>13</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Display Rate</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="DisplayRate">
    <property name="geometry">
     <rect>
      <x>125</x>
      <y>240</y>
      <width>49</width>
      <height>24</height>
     </rect>
    </property>
    <property name="maximum">
     <number>500</number>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="LengthPenalty">
    <property name="geometry">
     <rect>
      <x>125</x>
      <y>60</y>
      <width>49</width>
      <height>24</height>
     </rect>
    </property>
    <property name="singleStep">
     <double>0.001000000000000</double>
    </property>
   </widget>
   <widget class="QLabel" name="label_11">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>111</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>13</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Length Penalty</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>210</y>
      <width>111</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>13</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Num of Childs</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="NumberOfChilds">
    <property name="geometry">
     <rect>
      <x>125</x>
      <y>210</y>
      <width>49</width>
      <height>24</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="Selection">
    <property name="geometry">
     <rect>
      <x>85</x>
      <y>280</y>
      <width>91</width>
      <height>26</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_13">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>280</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>13</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Selection</string>
    </property>
   </widget>
   <widget class="QComboBox" name="Parents">
    <property name="geometry">
     <rect>
      <x>85</x>
      <y>310</y>
      <width>91</width>
      <height>26</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_14">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>310</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>13</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Parents</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>60</y>
     <width>411</width>
     <height>291</height>
    </rect>
   </property>
   <property name="title">
    <string>Output</string>
   </property>
   <widget class="QPlainTextEdit" name="CodeView">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>391</width>
      <height>181</height>
     </rect>
    </property>
    <property name="plainText">
     <string/>
    </property>
   </widget>
   <widget class="QLineEdit" name="OutputView">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>220</y>
      <width>321</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="ResultLabel">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>220</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Result:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>260</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Iterations:</string>
    </property>
   </widget>
   <widget class="QLabel" name="Iterations">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>260</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>260</y>
      <width>80</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Max Score:</string>
    </property>
   </widget>
   <widget class="QLabel" name="MaxScore">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>260</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_3">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>0</y>
     <width>411</width>
     <height>61</height>
    </rect>
   </property>
   <property name="title">
    <string>Input</string>
   </property>
   <widget class="QLineEdit" name="InputText">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>30</y>
      <width>151</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="InputLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>25</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Enter your Text</string>
    </property>
   </widget>
   <widget class="QPushButton" name="StartButton">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>25</y>
      <width>71</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Start</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ResetButton">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>25</y>
      <width>71</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Reset</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
